### ISSUE TITLE
Clearlogo not working (local artwork) on OSD

### ISSUE THREAD

**Issue by simtaj** [croneter/PlexKodiConnect#1936]
Using PlexKodiConnect with direct paths and local artwork saved in movie folder. I am using skin Arctic Horizon 2.

When adding movie in Kodi I have only poster and fanart if I check artwork trough manage. If I go trough movie information-manage and choose refresh, then all local artwork shows and works on OSD. But after that all refreshed movies disappears from choosed item menu (Videos,library://video/Plex-1/1_all/,return) and coresponded widgets.

They stays only in menu item (Videos,library://video/movies/titles.xml/,return)

Author of skin said to me: 

“This sounds like a plex kodi connect problem. Tmdbhelper doesn't control your library or plex kodi connect.
You would need to speak with the PKC author about how to add extra artwork to your library without affecting the plex paths.
If the item in the library has artwork then tmdbhelper will use it. Otherwise it needs to do a lookup which depends on the item having correct metadata”

**Comment by simtaj** [comment]
Author of skin explains more:

Tmdbhelper does NOT modify your library metadata in anyway.

"Info - Manage Art" is the artwork in your library. If you have artwork set for clearlogo there then TMDbhelper will use it

"tmdbhelper option - manage" is the artwork tmdbhelper has looked up online. This artwork is only used if the local item doesn't have artwork.

If your local item sets a clearlogo that doesn't exist then tmdbhelper will not have art. It only cares about whether a filename is listed, it doesn't read the artwork to check it exists.

**Comment by simtaj** [comment]
I tried today with offical Estaury skin and made totaly reset the kodi database from PKC. And I get the same result. No clearlogo, cleart…I get it only if I manual choose art, or refresh every single movie.

So it has to problem with PKC, because is on diffrent skin the same.

**Comment by simtaj** [comment]
@croneter 
with author of skin we came to conclusion that PKC need enhacement for this to work. Isi it possible to work on it ?
Please look at comments below:

Github fromjurialmunkey:    [](https://github.com/jurialmunkey/plugin.video.themoviedb.helper/issues/1021)




“**Also note that you still need to add the local artwork to your library. Kodi won't magically add it for you simply because you put it in a folder -- you need to add it to the library either manually in manage artwork, refresh the individual local information, or using a tool to refresh your whole library.
I think this is the main problem. There's nowhere I can add or mark that KODI would know that I have local artwork.
I can check (click) every single artwork item in info/manage artwork and then it accept it and show it. But is too many work for every single movie.”**

_““But I cannot use refresh artwork in info/manage artwork because the movie in PKC library after that disappears. With refresh Kodi not only refreshes artwork, but obviously changes path to Kodi library instead to leave it in PKC library.

Am I thinking right ?””_


“**“simtaj- yes spot on.

You would need to go through and manually add each one since you can't do a refresh or a rescan like you would with a standard local library.

Since PKC builds the library for you rather than a scraper, PKC would need to add an option to recognise local artwork as part of that process.

Those are your only options for local artwork. Tmdbhelper will only recognise local art added to the library.

If local art in PKC is not feasible thn you should use the online lookup feature in PKC. It is better to set artwork closer to the source as that will give you better control over your artwork and allow you to change it via the info dialog rather than rely on tmdbhelper.

Also note Tmdbhelper never downloads the artwork it looks up - it's only the url it saves - so it's not like you will be using more space by also having PKC do an online lookup (other than the fact you already have local artwork saved too). The artwork from a url gets downloaded on demand by kodi when it wants to display it on screen.””**

**Comment by croneter** [comment]
Yes, you're correct. PKC would need to have an additional function to add artwork that exists locally. This PKC feature does not exist yet. 

To be honest, this has very low priority for me.

**Comment by simtaj** [comment]
Thank you for your reply. 
Sorry to hear that. For me local artwork is much better. I can have it when I am offline and better managment, because I use Ember Media manager and I can decided witch artwork is right for movie/show. I have a lot of non hollywood movies and shows and here comes a lot of time problem with not proper artwork.

**Comment by HannesJo0139** [comment]
I wonder if this is the reason for my issue here
https://github.com/jurialmunkey/skin.arctic.fuse/issues/1079

**Comment by croneter** [comment]
> So if I get it correctly I will never have Clearlogos or so in my Kodi db? I wonder if this is the reason for my issue here. [jurialmunkey/skin.arctic.fuse#1079](https://github.com/jurialmunkey/skin.arctic.fuse/issues/1079)

No @HannesJo0139 I don't think so

**Comment by HannesJo0139** [comment]
@croneter Yeah I also think it is TMDb helper cache related somehow. Thank you for your quick reply.
